MuvrKit/MKSensorData.swift

```diff
@@ -158,6 +158,13 @@ public struct MKSensorData {
         return try MKSensorData(types: types, start: start + offset, samplesPerSecond: samplesPerSecond, samples: Array(data))
     }
     
+    public func sliceByCSVRow(from from: Int, to: Int) throws -> MKSensorData {
```

Comment:
Hmm... That definitely doesn't belong here. If you must, it may be acceptable to create a new extension, ``MKSensorData+CSV.swift`` and add the code there. But then why should this code be included in the MuvrKit framework when it is not used in our phone or watch apps. I think it'd be better to have an extension in the Playground itself.